server/lib/email.js

```diff
@@ -312,11 +314,23 @@ const generateEmailFromTemplate = (template, recipient, data = {}, options = {})
 /*
  * Given a template, recipient and data, generates email and sends it.
  */
-const generateEmailFromTemplateAndSend = (template, recipient, data, options = {}) => {
+const generateEmailFromTemplateAndSend = async (template, recipient, data, options = {}) => {
   if (!recipient) {
     logger.info(`Email with template '${template}' not sent. No recipient.`);
     return;
   }
+
+  // Skip active email notification check for unit test
```

Comment:
Let's maybe extract that in a function (ie: `isNotificationActive(template, data.collective.id, data.user.id)` returning a boolean.